search_service/proxy/elasticsearch.py

```diff
@@ -614,3 +615,48 @@ def delete_document(self, *, data: List[str], index: str) -> str:
             return ''
 
         return self._delete_document_helper(data=data, index=index)
+
+    @timer_with_counter
+    def fetch_dashboard_search_results(self, *,
+                                       query_term: str,
+                                       page_index: int = 0,
+                                       index: str = '') -> SearchResult:
+        """
+        Fetch dashboard search result with fuzzy search
+
+        :param query_term:
+        :param page_index:
+        :param index:
+        :return:
+        """
+        current_index = index if index else \
+            current_app.config.get(config.ELASTICSEARCH_INDEX_KEY, DEFAULT_ES_INDEX)
+
+        if not query_term:
+            # return empty result for blank query term
+            return SearchResult(total_results=0, results=[])
+        s = Search(using=self.elasticsearch, index=current_index)
+
+        query_name = {
+            "function_score": {
+                "query": {
+                    "multi_match": {
+                        "query": query_term,
+                        "fields": ["name.raw^30",
```

Comment:
s/30/75 ? Just as what we did on table recently?

Reply:
update